frontend/src/utils.js

```diff
@@ -70,3 +70,20 @@ export function getStockLogoUrl(ticker) {
     stockLogoUrl.searchParams.append("token", import.meta.env.VITE_FINNHUB);
     return stockLogoUrl.href;
 }
+
+export function getMarketNewsUrl() {
+    const marketNewsUrl = new URL("https://finnhub.io/api/v1/news");
+    marketNewsUrl.searchParams.append("category", "forex");
+    marketNewsUrl.searchParams.append("minId", "10");
+    marketNewsUrl.searchParams.append("token", import.meta.env.VITE_FINNHUB);
+    return marketNewsUrl.href;
+}
+
+export function getStockNewsUrl(currentStock) {
+    const stockNewsUrl = new URL("https://finnhub.io/api/v1/company-news");
+    stockNewsUrl.searchParams.append("symbol", currentStock);
+    stockNewsUrl.searchParams.append("from", "2023-07-01");
+    stockNewsUrl.searchParams.append("to", getCurrentDate());
+    stockNewsUrl.searchParams.append("token", import.meta.env.VITE_FINNHUB);
+    return stockNewsUrl.href;
```

Comment:
nice job remembering to use the `URL` API! definitely makes handling multiple parameters more manageable